src/rpc/misc.cpp

```diff
@@ -586,16 +588,31 @@ UniValue logging(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() > 2) {
         throw std::runtime_error(
-            "logging [include,...] <exclude>\n"
+            "logging (<include> <exclude>)\n"
```

Comment:
The standard is to have put space after `(` and before `)` for optional args, i.e. `( <include> <exclude> )`